src/main/java/frc/robot/subsystems/triggerSubsystem/TriggerCommandFactory.java

```diff
@@ -32,6 +35,8 @@ public void setDefaultCommand(Command defaultCommand) {
   private Command buildCommand(double speed) {
     if (subsystem == null) return Commands.none();
     return new StartEndCommand(
-        () -> subsystem.setSpeed(speed), () -> subsystem.setSpeed(0), subsystem);
+            () -> subsystem.setSpeed(speed), () -> subsystem.setSpeed(0), subsystem)
+        .withName("buildCommand")
+        .asProxy();
```

Comment:
Since this one is private (i.e. used to build a bigger Command composition, don't want proxy or name added.